### PR TITLE
fix: MQTT Connector

### PR DESCRIPTION
**Pull request by joeylimd**
Fixes: #1717

### CODE REVIEW

File: app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+    "id": "paho",
```
Comment (lburgazzoli) [inline app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json]: I would name both the file and the id as  `mqtt` to reflet the connector name, paho

File: app/connectors/connectors/mqtt/src/main/java/io/syndesis/connector/mqtt/MqttVerifierAutoConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import org.springframework.context.annotation.Lazy;
+import org.springframework.context.annotation.Scope;
+
+import io.syndesis.verifier.api.Verifier;
+
+@Configuration
+@ConditionalOnProperty(prefix = "io.syndesis.connector.verifier", name = "enabled")
+public class MqttVerifierAutoConfiguration {
+
+    @Bean("paho")
```
Comment (lburgazzoli) [inline app/connectors/connectors/mqtt/src/main/java/io/syndesis/connector/mqtt/MqttVerifierAutoConfiguration.java]: I would use `mqtt`

File: app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    },
+    "actions": [
+        {
+            "name": "Publish",
+            "description": "Publish a message to a MQTT Broker",
+            "id": "io.syndesis:paho-publish-connector",
+            "pattern": "To",
+            "actionType": "connector",
+            "descriptor": {
+                "componentScheme": "paho",
```
Comment (lburgazzoli) [inline app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json]: if `componentScheme` is set as connector level, it is not needed to define it again

File: app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            }
+        },
+        {
+            "name": "Subscribe",
+            "description": "Receive a message from MQTT Broker",
+            "id": "io.syndesis:mqtt-subscribe-connector",
+            "pattern": "From",
+            "actionType": "connector",
+            "descriptor": {
+                "componentScheme": "paho",
```
Comment (lburgazzoli) [inline app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json]: if `componentScheme` is set as connector level, it is not needed to define it again

File: app/connectors/pom.xml [hunk 5]
```diff
@@ -78,6 +78,7 @@
     <module>connectors/jms/amqp-connector</module>
     <module>connectors/odata</module>
 	<module>connectors/ftp</module>
+	  <module>connectors/mqtt</module>
```
Comment (lburgazzoli) [inline app/connectors/pom.xml]: don't use tabs

File: app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json (hunk 1 above)
Comment (joeylimd) [reply app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json]: Done, please review again. Thanks, Luca.

File: app/connectors/connectors/mqtt/src/main/java/io/syndesis/connector/mqtt/MqttVerifierAutoConfiguration.java (hunk 2 above)
Comment (joeylimd) [reply app/connectors/connectors/mqtt/src/main/java/io/syndesis/connector/mqtt/MqttVerifierAutoConfiguration.java]: Done, please review again. Thanks, Luca.

File: app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json (hunk 3 above)
Comment (joeylimd) [reply app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json]: Done, please review again. Thanks, Luca.

File: app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json (hunk 4 above)
Comment (joeylimd) [reply app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json]: Done, please review again. Thanks, Luca.

File: app/connectors/pom.xml (hunk 5 above)
Comment (joeylimd) [reply app/connectors/pom.xml]: Done, please review again. Thanks, Luca.

**Pull request merged**